sample/src_polymer/elements/scene-perception-pages/scene-perception-page.html

```diff
@@ -212,8 +212,6 @@
   
   <script src='../../libs/three.js'></script>
   <script src='../../libs/TrackballControls.js'></script>
-  <script src='../../libs/stats.min.js'></script>
-  <script src='../../scripts/common/utils.js'></script>
   <script src='../../scripts/common/spMath.js'></script>
```

Comment:
We'd better also move `scripts/common/spMath.js` into `common` then we can get rid of `scripts/common`?